models/aragon/arbitrum/aragon_arbitrum_schema.yml

```diff
@@ -0,0 +1,68 @@
+version: 2
+
+models:
+  - name: aragon_arbitrum_app_dao_addresses
+    meta:
+      blockchain: arbitrum
+      project: aragon
+      contributors: Henrystats 
+    config:
+      tags: [ 'arbitrum', 'dao', 'aragon', 'dao creator tools', 'Henrystats', 'Aragon app']
+    description: >
+      wallet addresses of daos created on aragon app
+    tests:
+      - dbt_utils.unique_combination_of_columns:
```

Comment:
the test columns don't match the model columns, plz update one or the other to match and ensure uniquness